add_support_for_multiple_dockers_in_lint (#2783)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

## Unreleased

> * added support for multiple dockers separated by comma for native docker linter.

Reviewer: ```suggestion
* added support for running **lint** in multiple native-docker images.
```
